components/html-block/html-block.js

```diff
@@ -178,24 +175,31 @@ class HtmlBlock extends LitElement {
 		super.connectedCallback();
 		if (this._contextObserverController) this._contextObserverController.hostConnected();
 
-		if (!this._templateObserver || this.noDeferredRendering) return;
+		if (!this._contentObserver || this.noDeferredRendering) return;
 
 		const template = this._findSlottedElement('TEMPLATE');
-		if (template) this._templateObserver.observe(template.content, { attributes: true, childList: true, subtree: true });
+		if (template) this._contentObserver.observe(template.content, { attributes: true, childList: true, subtree: true });
```

Comment:
Is this just here for backwards compatibility, and will be removed once consumers are switched?  Currently, it looks like `assignedNodes `needs to be resolved at least twice during initial render.

Reply:
Yes, this is strictly for backwards compatibility and can go away as soon as consumers have been updated. I acknowledge that it's sort of gross as is (because of that double rendering). I figured it wasn't worth splitting out the `assignedNodes` call just for this brief compatibility window, but if there are significant concerns I can also do that.

Reply:
I might just check to see how many times that resolution is going to be required even after the template clean-up.  Otherwise I think it is fine.

Reply:
Sounds good. It's been a bit since I wrote the code so I'll double check, but my goal was to align things so that the number of resolutions is unchanged from the templated version.